### DISCUSSION TITLE
nerdctl doesnt display images / containers

### DISCUSSION THREAD

**Discussion by shnal12** [containerd/nerdctl#2406] [Q&A] [answered]
Hi, 
I would like to save the running containers  as tarfile for use in docker in other VM. 
when I use crictl command with -c  <path>/crictl.yaml pods ls 
I get output  of pods 
But with nerdctl , its empty. Let me know how to proide config  path of crictl like above

**Comment by fahedouch** [comment, marked as answer]
please check [nerdctl config documentation](https://github.com/containerd/nerdctl/blob/main/docs/config.md#file-path)

contrary to `crictl`, `nerdctl` do not display pods but it display k8s containers by setting  `k8s.io` namespace : `# nerdctl --namespace k8s.io ps -a` ([see](https://github.com/containerd/nerdctl#debugging-kubernetes))
